File: backend/insurance_engine/InsuranceEngine/InsuranceEngine.Domain/Entities/ContactInfo.cs
using System.ComponentModel.DataAnnotations;

namespace InsuranceEngine.Domain.Entities;

public class ContactInfo
{
    [MaxLength(50)]
    public string? MobileNumber { get; set; }

    [MaxLength(255)]
    public string? Email { get; set; }

    [MaxLength(50)]
    public string? AlternateMobile { get; set; }

    [MaxLength(50)]
    public string? Landline { get; set; }
}

File: sdks/insuretech-go-sdk/pkg/models/contact_info.go
package models


// ContactInfo represents a contact_info
type ContactInfo struct {
	AlternateMobile string `json:"alternate_mobile,omitempty"`
	Email string `json:"email,omitempty"`
	Landline string `json:"landline,omitempty"`
	MobileNumber string `json:"mobile_number,omitempty"`
}
